mbof/views.py

```diff
@@ -26,6 +27,15 @@ class EventViewSet(viewsets.ModelViewSet):
     """
     queryset = Event.objects.all().order_by('postingTime')
     serializer_class = EventSerializer
+    
+
+class CurrentEventViewSet(EventViewSet):
+    """
+    API endpoint that allows users to be viewed or edited.
+    """
+    current_time = timezone.now()
+    queryset = Event.objects.filter(startTime__lte=current_time).filter(endTime__gte=current_time)\
```

Comment:
I think both of these conditions can be combined in one call to `filter()`.  Maybe:

```python
queryset = Event.objects.filter(startTime__lte=current_time, endTime__gte=current_time).order_by('postingTime')
```